examples/schoolyard.jl

```diff
@@ -26,11 +26,9 @@ using SimpleWeightedGraphs: SimpleWeightedDiGraph # will make social network
 using SparseArrays: findnz                        # for social network connections
 using Random: MersenneTwister                     # reproducibility
 
-# And create a very simple agent without any extra properties
-mutable struct Student <: AbstractAgent
-    id::Int
-    pos::Tuple{Float64,Float64}
-end
+# And create an alias to `ContinuousAgent{2}`,
+# as our agents don't need additional properties.
+Student = ContinuousAgent{2}
```

Comment:
This should be `const`, since it shouldn't be reassigned, and non-const globals are typically slower.

Reply:
Yes, but this is not true for type aliasing.